README.md

```diff
@@ -1,23 +1,41 @@
-Sonata Doctrine Extensions
-==========================
+# Sonata Doctrine Extensions
 
-[![Build Status](https://travis-ci.org/sonata-project/sonata-doctrine-extensions.svg?branch=master)](https://travis-ci.org/sonata-project/sonata-doctrine-extensions)
+Symfony doctrine extensions
```

Comment:
Symfony?

Reply:
Yes, it's intended:
https://github.com/sonata-project/form-extensions#sonata-form-extensions
https://github.com/sonata-project/SonataNewsBundle#sonata-news-bundle